cypress/test/NewsletterSubscribe.cy.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { mount } from 'cypress/react';
+import NewsletterSubscribe from '../../components/NewsletterSubscribe';
+describe('NewsletterSubscribe Component', () => {
+  it('renders the component with default props', () => {
+    mount(<NewsletterSubscribe />);
+
+    cy.get('div.p-8.text-center').should('exist');
+    cy.contains('Subscribe to our newsletter').should('exist');
```

Comment:
declare all the hardcode text in a constant variable, and use it here
const NewsletterConstants = [
"sentence1",
 "sentence2"
]
And use Newsletter[0] like that, so the code becomes maintainable, when the text changes.